packages/cloud/src/TelemetryClient.ts

```diff
@@ -20,7 +20,8 @@ export class TelemetryClient extends BaseTelemetryClient {
 		)
 	}
 
-	private async fetch(path: string, options: RequestInit) {
+	// kilocode_change: unused params
+	private async fetch(_path: string, _options: RequestInit) {
```

Comment:
wouldn't it be easier to ignore the lint warning with block exceptions? Then the line itself isnt change and the merge might be easier